oe_corporate_blocks.post_update.php

```diff
@@ -78,7 +82,9 @@ function _oe_corporate_blocks_import_corporate_links(): void {
 
   foreach ($configs as $config) {
     $config_storage->write($config, $source->read($config));
-    $config_factory->getEditable($config)->save();
+    // All records in cache.config have empty cache tags column. In this case,
+    // we have to remove cache by cache id.
+    \Drupal::cache('config')->delete($config);
```

Comment:
interesting but since the render changes, you also need to flush that after this loop.
`\Drupal::service('cache.render')->invalidateAll();`